src/utils/feature-flag-utils.ts

```diff
@@ -14,7 +14,8 @@ const defaultFlagSet: LDFlagSet = {
   'conversion-ui-enabled': false,
   'restoration-ui-enabled': false,
   'sentry-enable': false, // by default, no sentry logs
-  'supported-correction-entities': []
+  'supported-correction-entities': [],
+  'supported-alteration-change-business-types': ['BEN', 'BC', 'CC', 'ULC']
```

Comment:
Don't merge this. This value must not cause the app to wrong incorrectly if LaunchDarkly can't be reached.

Reply:
I'll set it back to 'BEN', 'BC', that's the default today if that's ok?

Reply:
I think this should be `[]`.

You can set the dev LD key in your .env file to get the actual flags (which are OK in dev, right).

Reply:
Yup, the problem was unit tests here though

Reply:
Oh, then mock the FF getter.

Reply:
I mocked it in setup, there were a few groups of unit tests that relied on this